homeassistant/components/drop_connect/strings.json

```diff
@@ -31,6 +31,7 @@
       "pending_notification": { "name": "Notification unread" },
       "reserve_in_use": { "name": "Reserve capacity in use" },
       "salt": { "name": "Salt low" },
+      "alert_sensor": { "name": "Sensor" },
```

Comment:
Please keep the keys alphabetically sorted, it's annoying to have to go through the entire map to check if something is there or not.

Reply:
Addressed in https://github.com/home-assistant/core/pull/117867/commits/0abb8e1bd38c47ed7448030452b89cf833ae26e6